token-metadata/program/src/processor/bubblegum/bubblegum_set_collection_size.rs

```diff
@@ -29,12 +29,6 @@ pub fn bubblegum_set_collection_size(
     let collection_mint_account_info = next_account_info(account_info_iter)?;
     let bubblegum_signer_info = next_account_info(account_info_iter)?;
 
-    let delegated_collection_auth_opt = if accounts.len() == 5 {
-        Some(next_account_info(account_info_iter)?)
-    } else {
-        None
-    };
-
     if !BUBBLEGUM_ACTIVATED {
         return Err(MetadataError::InvalidOperation.into());
     }
```

Comment:
Should we add a comment indicating that collection authority is checked in Bubblegum or is that obvious?